artemis-distribution/src/main/resources/licenses/bin/LICENSE

```diff
@@ -432,4 +432,46 @@ For Underscore:
 ==============================================================================
 This product bundles Underscore, which is available under a
 "MIT" license.  For details, see
-https://github.com/jashkenas/underscore/blob/master/LICENSE
\ No newline at end of file
+https://github.com/jashkenas/underscore/blob/master/LICENSE
+
+==============================================================================
+For jakarta.activation-api:
+==============================================================================
+This product bundles Underscore, which is available under the
+Eclipse Distribution License (EDL) 1.0. For details, see licenses/EDL-1.0.txt
+
+==============================================================================
+For jakarta.inject-api:
+==============================================================================
+This product bundles Underscore, which is available under the
+Apache License Version 2.0. For details, see licenses/EPL-2.0.txt
```

Comment:
This says "Apache License Version 2.0" but then links to EPL-2.0. Which is it?

Note if it is ALv2 you dont need to mention it at all since its already covered.

Reply:
removed